meta/saiserialize.h

```diff
@@ -415,6 +415,84 @@ int sai_deserialize_mac(
         _In_ const char *buffer,
         _Out_ sai_mac_t mac);
 
+/**
+ * @brief Serialize macsec_sak.
+ *
+ * @param[out] buffer Output buffer for serialized value.
+ * @param[in] sak The macsec_sak to be serialized.
+ *
+ * @return Number of characters written to buffer excluding '\0',
+ * or #SAI_SERIALIZE_ERROR on error.
+ */
+int sai_serialize_macsec_sak(
```

Comment:
those functions looks great, can you also add tests for those in meta/saiserializetest.c ? most of user declared serialization/serialization functions are declared there to make sure they works correctly